pkg/boot/v2_guid_map.go

```diff
@@ -0,0 +1,73 @@
+package boot
+
+// This is to solve the service/plan IDs conflict between Meta Azure Service
+//   Broker and this Open Service Broker for Azure to allow their co-existence.
+// nolint: lll
+func getV2GuidMap() map[string]string {
+	return map[string]string{
```

Comment:
What's the meaning of the key and the value of the map?

Reply:
Mentioned on top:
>Add a set of V2 service/plan GUIDs. It is for the co-existence of OSBA and MASB as OSBA has some GUID conflict with MASB. It brings a smooth migration path to users. Once the option in the manifest is enabled, a flag record would be wrote to the broker back store. And this OSBA instance would use V2 GUID forever and never care about the option in manifest.

Reply:
... So the key is new generated uuid and the value is old id used by maba?

Reply:
The keys are original OSBA GUIDs. But some of them are conflict to MASB. The values are totally new.

Reply:
Got it. Thanks.